packages/fxa-settings/src/pages/Signin/Signin/index.tsx

```diff
@@ -0,0 +1,248 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
How do you / @vpomerleau feel about this component being `pages/Signin/index.tsx` rather than `pages/Signin/Signin/index.tsx`, and same for `ResetPassword`? `Signin/Signin` / `ResetPassword/ResetPassword` feels a little awkward to me personally and it'd match what we've done with `Settings/`.

Reply:
Oh yeah, totally! I hadn't thought of doing it that way but it would definitely be less redundant.